Jenkinsfile

```diff
@@ -125,22 +121,136 @@ pipeline {
               '''
             }
           }
+
+          post {
+            always {
+              archiveArtifacts "results/**/*"
+              junit "results/results.xml"
+              withCredentials([
+                [
+                  $class: "AmazonWebServicesCredentialsBinding",
+                  credentialsId: "f40eebe0-f9aa-4336-b460-b2c4d7876fde",
+                  accessKeyVariable: "AWS_ACCESS_KEY_ID",
+                  secretKeyVariable: "AWS_SECRET_ACCESS_KEY"
+                ]
+              ]) {
+                sh "./system-tests/_scripts/delete-cluster.sh"
+              }
+            }
+          }
+        }
+      }
+    }
+
+    stage("System Test previous versions") {
```

Comment:
this is a first step, next one will be having these running in parallel and on every PR 👍